addons/l10n_ch/models/res_bank.py

```diff
@@ -242,6 +242,8 @@ def _get_qr_code_generation_params(self, qr_method, amount, currency, debtor_par
                 'quiet': 1,
                 'mask': 'ch_cross',
                 'value': '\n'.join(self._get_qr_vals(qr_method, amount, currency, debtor_partner, free_communication, structured_communication)),
+                # Swiss QR code requires Error Correction Level = 'M' by specification
+                'kwargs': {'barLevel': 'M'}
```

Comment:
`report_barcode` takes actual kwargs so I don't think that's going to work.

Reply:
It was working, but the whole flow is better with the actual `**kwargs`, I do agree.